client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -93,7 +95,27 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 		}
 	};
 
+	receiveMessage = ( event ) => {
+		if ( event.origin && event.origin !== location.origin ) {
+			return;
+		}
+
+		const data = 'string' === typeof event.data
+			? JSON.parse( event.data )
+			: event.data;
```

Comment:
Huh? Not sure I follow, since `event.data` is already an object. I discovered parsing wasn't necessary by actually trying to parse it and then realizing the `catch` handler fired every time. I was like, 'Calypso, `JSON.parse( event.data )`', and Calypso was all like `Unexpected token o in JSON at position 1`. So buh bye `JSON.parse`!